### PR TITLE
add quality setting; allow png for lossless compression

### PR DESCRIPTION
**Pull request by machenmusik**
#1579 removed WebRTC from the viewer in favor of JPEG image compression.
However, JPEG compression is lossy, and JPEG quality was hardcoded (75, down from the default of 95).
This can cause noticeable viewer artifacts.

This PR adds: 
- by default, viewer JPEG image format with 90 quality
- configurable viewer PNG compression and JPEG quality settings

### CODE REVIEW

**Comment by LownyCGI** [comment]
Hello, @machenmusik basically we can add support for openEXR aswell, right? it's more color precise.

**Comment by machenmusik** [comment]
> Hello, @machenmusik basically we can add support for openEXR aswell, right? it's more color precise.

In theory, sure - although I don't think Pillow supports OpenEXR directly, and neither do web browsers.  

The viewer tends to change requested image sizes drastically during interaction, as well.  It might be better to integrate the ability to render OpenEXR, for higher quality.

I would suggest working on OpenEXR support as separate PRs, as the complexity could be significantly higher than this simple one.

File: nerfstudio/configs/base_config.py [hunk 1]
```diff
@@ -148,3 +151,9 @@ class ViewerConfig(PrintableConfig):
     actually used in training/evaluation. If -1, display all."""
     quit_on_train_completion: bool = False
     """Whether to kill the training job when it has completed. Note this will stop rendering in the viewer."""
+    image_format: Literal["jpeg", "png"] = "png"
```
Comment (tancik) [inline nerfstudio/configs/base_config.py:154]: I think the default should stay jpeg, but we can bump the quality to 85 or 90

Comment (machenmusik) [reply nerfstudio/configs/base_config.py:154]: For remote usage, I can see using JPEG by default, but would be good to increase quality.

For local usage where not bandwidth constrained, PNG looks much nicer IMO.

I wonder whether the default should be "auto" or something.

**Review by tancik — APPROVED** [review]
LGTM

**Pull request merged**